### PR TITLE
Mvt inline workers

### PR DESCRIPTION
**Pull request by juandjara**
For #14 

#### Background
The comments in #15 

#### Change List
- Merge with latest uber's `master` branch as of `2 Dec 2019`
- Use `webworkify-webpack` to bundle inline the MVT workers. I've taken example from other places in the DeckGL repo where they are using this same module and adapt their approach to bundling workers inline.

### CODE REVIEW

File: modules/geo-layers/src/mvt-tile-layer/workers/worker.js [hunk 1]
```diff
@@ -1,7 +1,12 @@
 import { MVTWorker } from './mvt-worker';
 
-const mvtWorkerInstance = new MVTWorker();
+/* global self */
 
-onmessage = function (event) {
-    return mvtWorkerInstance.onMessage(event);
-};
+export default self => {
+  const mvtWorkerInstance = new MVTWorker();
```
Comment (jesusbotella) [inline modules/geo-layers/src/mvt-tile-layer/workers/worker.js]: Why do you think that it's better to create an instance of the worker every time?

File: modules/geo-layers/src/mvt-tile-layer/workers/worker.js [hunk 2]
```diff
@@ -1,7 +1,12 @@
...
+  self.onmessage = ev => {
+    mvtWorkerInstance.processEvent(ev).then(data => {
```
Comment (jesusbotella) [inline modules/geo-layers/src/mvt-tile-layer/workers/worker.js]: I think it's better to leave that function within Worker's code, and then if `self` is a global variable use it within the class or pass it through the method invocation.

File: modules/geo-layers/src/mvt-tile-layer/workers/worker.js (hunk 1 above)
Comment (juandjara) [reply modules/geo-layers/src/mvt-tile-layer/workers/worker.js]: It's just some experiments I did while debugging an error where the worker was not posting the message with the mvt data back to the original thread. I thought it had to with worker and webpack scopes but I checked again and it's working ok the way you say too.

File: modules/geo-layers/src/mvt-tile-layer/workers/worker.js (hunk 2 above)
Comment (juandjara) [reply modules/geo-layers/src/mvt-tile-layer/workers/worker.js]: This has the same reason than the above comment

**Comment by juandjara** [comment]
Ok I've uploaded some fixes to make the new changes look closer to the original code

**Pull request closed without merging**